lib/ja_serializer/dsl.ex

```diff
@@ -0,0 +1,287 @@
+defmodule JaSerializer.DSL do
+  alias JaSerializer.Relationship.{HasMany,HasOne}
```

Comment:
Ha, it is 1.2 only, I just removed it to maintain backwards compatibility. Same as doing two aliases.